### PR TITLE
added PaperReceived to Waiver io

### PR DESCRIPTION
**Pull request by CritiqalPete**
hoping this change is sufficient to support this additional field on the Reg form(s)

### CODE REVIEW

**Comment by CritiqalPete** [comment]
to verify we need to send a test waiver to this API along with signed_paper_form_received__C (TRUE)

**Comment by CritiqalPete** [comment]
@AleksandrMolchagin review if you're ready.

File: src/main/mule/waiver.xml
Comment (AleksandrMolchagin) [inline src/main/mule/waiver.xml:1]: DataWeave is case-sensitive and adheres to JS conventions. 'FALSE' should be written as 'false' in lowercase.

Comment (AleksandrMolchagin) [reply src/main/mule/waiver.xml:1]: @CritiqalPete, the Production database doesn't have the object 'Waiver' (or 'Waiver__c'). Is it currently only in the Sandbox database? I couldn't access Sandbox through https://workbench.developerforce.com/

Comment (AleksandrMolchagin) [reply src/main/mule/waiver.xml:1]: Pete and I did a further investigation. The Production database DOES include the object 'Waiver' (or 'Waiver__c'). I had a problem with access token.

**Review by AleksandrMolchagin — APPROVED** [review]

**Pull request merged**